README.md

```diff
@@ -24,93 +37,127 @@ Add two new maven repository destination in the project-level build.gradle repos
 
         }
     }
-  ```
-
-Add two dependency imports in the application build.gradle dependency list:
-
-  ``` java
+```
+Add two dependencies in the application ```build.gradle``` :
+```gradle
     implementation 'com.veriff:veriff-library:2.3.0'
     implementation 'io.probity.sdk:collector:1.0.0'
-  ```
+```
+## Starting the verification flow
 
-#### Veriff SDK 1.*
-  Veriff SDK versions 1.* are no longer being supported, please integrate version 2.* instead.
+The verification process must be launched inside the vendor specific Activity class.
+The activity is aware of the current customer, who is either logged in via the Vendor system or who
+is identified other way with unique system pass. The vendor must be able to determine the customer
+later, if application ends/ or returns. It depends entirely on the Vendor business logic.
 
-  To integrate Veriff into a new project, follow the previous section
-  To upgrade from 1.* to 2.* refer to section [on upgrading Veriff](#upgrading-veriff)
+Every Veriff SDK session is unique for a client. The session expires after 7 days automatically,
+but is valid until then. After the verification data is uploaded, the SDK v1.0 does not wait for the
+final verification result (async). The SDK v1.0 only notifies whether the verification data upload is successful or not.
 
-## Using the Veriff SDK
+In vendor activity class define the result code and initialize the SDK and launch the verification
+flow as below:
 
-### 2. Initializing SDK in project code
+| Parameters   | Description                                                  |
+| ------------ | ------------------------------------------------------------ |
+| baseUrl      | 'https://staging.veriff.me/v1/' for testing and 'https://magic.veriff.me/v1/' for production |
```

Comment:
:-1: the URL's shouldn't include `/v1/`, we actually actively discourage that at runtime by emitting a warning in logcat